libs/postgresql/Makefile

```diff
@@ -222,6 +222,11 @@ define Build/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/usr/include/postgresql $(1)/usr/include/
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpq.{a,so*} $(1)/usr/lib/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/pg_config $(1)/usr/bin/
```

Comment:
`pg_config` is an executable compiled for the target, it cannot just go into staging_dir/host as it typically cannot be executed on the host. We also already install a host-compiled version of it in `Host/Install`, so no need to overwrite it with the target-compiled version in InstallDev.